### PR TITLE
[Review] Closure date #291

### PR DESCRIPTION
**Pull request by dhilipsiva**
Dont merge this yet! Closure date #291

[Review on Reviewable](https://reviewable.io/reviews/pythonindia/junction/307)

### CODE REVIEW

**Comment by dhilipsiva** [comment]
@kracekumar Tagged it as Review because not sure if the modified `_get_proposal_section_choices` will affect ant other part of the code. 
Will send a seperate request for #302

File: junction/proposals/models.py [hunk 1]
```diff
@@ -27,6 +29,8 @@ class ProposalSection(AuditModel):
     description = models.TextField(default="")
     active = models.BooleanField(default=True, verbose_name="Is Active?")
     conferences = models.ManyToManyField(to=Conference, related_name='proposal_sections')
+    start_date = models.DateField(default=datetime.now, verbose_name="Start Date")
+    end_date = models.DateField(default=datetime.now, verbose_name="End Date")
```
Comment (kracekumar) [inline junction/proposals/models.py]: We should these two fields in admin page as well right https://github.com/pythonindia/junction/blob/master/junction/proposals/admin.py#L25 ?

File: junction/proposals/views.py [hunk 2]
```diff
@@ -123,7 +124,7 @@ def list_proposals(request, conference_slug):
 def create_proposal(request, conference_slug):
     conference = get_object_or_404(Conference, slug=conference_slug)
     if request.method == 'GET':
-        if conference.status != 1:
+        if conference.status != 1 and conference.proposal_sections.filter(end_date__gt=now()).count() == 0:
```
Comment (kracekumar) [inline junction/proposals/views.py]: We can remove the check for conference status.

File: junction/proposals/models.py (hunk 1 above)
Comment (dhilipsiva) [inline junction/proposals/models.py]: I dont see why not. I just added them.

**Comment by dhilipsiva** [comment]
@kracekumar Is this okay now?

**Comment by kracekumar** [comment]
:+1:

**Comment by kracekumar** [comment]
If there are any missing cases, will create new issue.

**Pull request merged**